spec/presenters/two_factor_options_presenter_spec.rb

```diff
@@ -140,4 +140,19 @@
       end
     end
   end
+
+  describe '#show_cancel_return_to_sp?' do
+    context 'phishing resistant required to add additonal mfa' do
+      let(:presenter) do
+        described_class.new(
+          user_agent: user_agent, user: user_with_2fa,
+          phishing_resistant_required: true
```

Comment:
Nit: Stylistically I'd recommend either one per line or all on one line. I find it can be difficult to scan when there are different inconsistent numbers of properties between lines.

```suggestion
          user_agent: user_agent,
          user: user_with_2fa,
          phishing_resistant_required: true,
```